roles/docker/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+# Default values for docker role
+
+docker_version: 1.9.1
```

Comment:
@DivyaVavili 

just a clarification, is there a reason to use 1.9.1 version now that 1.10.1 is out? I can take care of bumping up this version if you don't see any issues.